modin/pandas/test/test_groupby.py

```diff
@@ -84,7 +84,7 @@ def test_mixed_dtypes_groupby(as_index):
     ]
 
     for by in by_values:
-        if by_values[0] == "col3":
+        if isinstance(by[0], str) and by[0] == "col3":
```

Comment:
why change? seems pointless

Reply:
This branch was never run before, because `by_values[0]` was comparised with string, but each value of `by_values` is tuple.

Reply:
oh, right. in this case I suggest to change it to
```python
if by[0] == "col3":
```
(there's no point for checking for `isinstance()`)

Reply:
There's a point to check this, because `by[0]` might be `Series` object (see the possible values of `by_values`). In that case comparison result is Series, but not bool value and we get an error.